worldedit-bukkit/src/main/java/com/sk89q/bukkit/util/CommandsManagerRegistration.java

```diff
@@ -32,7 +32,6 @@
 import java.util.Map;
 
 @Deprecated
-@SuppressWarnings("deprecation")
```

Comment:
IIRC there was a warning in `javac` suppressed by this. Is this no longer necessary?

Reply:
None appear to show - but AFAIK classes marked with `@Deprecated` do not show deprecation warnings anyway?